server/src/infra/graphql/query.rs

```diff
@@ -296,23 +297,23 @@ impl<Handler: BackendHandler> User<Handler> {
         self.attributes
             .iter()
             .find(|a| a.attribute.name.as_str() == "first_name")
-            .map(|a| a.attribute.value.unwrap())
+            .map(|a| a.attribute.value.as_str().unwrap_or(""))
             .unwrap_or("")
     }
 
     fn last_name(&self) -> &str {
         self.attributes
             .iter()
             .find(|a| a.attribute.name.as_str() == "last_name")
-            .map(|a| a.attribute.value.unwrap())
+            .map(|a| a.attribute.value.as_str().unwrap_or(""))
             .unwrap_or("")
     }
 
     fn avatar(&self) -> Option<String> {
         self.attributes
             .iter()
             .find(|a| a.attribute.name.as_str() == "avatar")
-            .map(|a| String::from(&a.attribute.value.unwrap::<JpegPhoto>()))
+            .map(|a| String::from(a.attribute.value.as_jpeg_photo().unwrap()))
```

Comment:
`.expect("Invalid JPEG returned by the DB")` maybe? Instead of unwrap

Reply:
Done